docs/usage/configuration-options.md

```diff
@@ -466,6 +466,18 @@ If configured to `true`, then dependency extraction will be done using the relev
 
 This option applies only to the `gradle` manager.
 
+## defaultRegistryUrls
```

Comment:
I'm still a bit fuzzy on _why_ I would need this feature. What it the problem that `defaultRegistryUrls` solves? _When_ would I need to use this feature?

Reply:
I added some clarification below. Most people won't need to use this, so it's not important for all to understand